tests/SessionTest.php

```diff
@@ -83,34 +101,10 @@ public function testRestart()
         $this->session->restart();
     }
 
-    public function testVisitWithoutRunningSession()
```

Comment:
But the test ensures that start is called:

````php
    public function testVisit()
    {
        $this->driver
            ->expects($this->once())
            ->method('start');

        $this->driver
            ->expects($this->once())
            ->method('visit')
            ->with($url = 'some_url');

        $this->session->visit($url);
    }
````

The test with isStarted is not contained in `testStartWithoutRunningSession` and `testStartWithRunningSession` as the logic is moved to the `start` method.